### ISSUE TITLE
训练模型无法复现结果

### ISSUE THREAD

**Issue by fupiao1998** [ltkong218/IFRNet#6]
您好，感谢您开源代码，代码写的很清晰，readme的可操作性也很强。但是我在尝试训练IFRNet_S的时候，发现我训练后的模型无法正确复现论文中的结果。我用了4张3090，完全按照readme中的命令进行的操作。最终只能得到34.45的PSNR。我的train.log如下，想问问可能出现的问题会在哪里，期待您的回复。
[train.log](https://github.com/ltkong218/IFRNet/files/8975261/train.log)

**Comment by ltkong218** [comment]
Is PSNR of IFRNet_S equals to 34.45 or 35.46 according to your experiment?

**Comment by fupiao1998** [comment]
Yes, follow the Readme, I generate the flow by liteflownet, and then perform training process by `python -m torch.distributed.launch --nproc_per_node=4 train_vimeo90k.py --world_size 4 --model_name 'IFRNet' --epochs 300 --batch_size 6 --lr_start 1e-4 --lr_end 1e-5`.
But finally, I only get the PSNR of 34.45 or 35.46.

**Comment by ltkong218** [comment]
In your provided <code>train.log</code> file, the <code>IFRNet_S</code> model has achieved PSNR of 35.46, why you say that you get the PSNR of 34.45?

**Comment by fupiao1998** [comment]
I apologize for my mistake, but the accuracy in the paper should be 35.59, which is a huge difference from the results I trained, and I think this is the problem we need to solve.

**Comment by fupiao1998** [comment]
In addition, in the log I provided, the optimal precision should be 35.46, which I incorrectly described as 34.45. I am sorry for my mistake.
But 35.46 is still 0.13dB away from 35.59 in the paper, so I wanted to find out what went wrong.

**Comment by ltkong218** [comment]
Some hyperparameters may should be adjusted for <code>IFRNet_S</code>. You can set <code>--lr_end 3e-5</code>, reduce the data augmentatioin probability of [random_resize](https://github.com/ltkong218/IFRNet/blob/1ee657dd4ae6c4b418faed05e347943d0223af47/datasets.py#L10) and [random_rotate](https://github.com/ltkong218/IFRNet/blob/1ee657dd4ae6c4b418faed05e347943d0223af47/datasets.py#L59) and set <code>crop_size=(256, 256)</code> in [random_crop](https://github.com/ltkong218/IFRNet/blob/1ee657dd4ae6c4b418faed05e347943d0223af47/datasets.py#L19). You should get the results of <code>IFRNet</code> in the paper by running provided training script.

**Comment by fupiao1998** [comment]
Thank you very much for your reply, I will try the way you said. Good luck with your future research, thank you again.
